src/main/java/cubicchunks/worldgen/GeneratorStage.java

```diff
@@ -23,27 +23,52 @@
  */
 package cubicchunks.worldgen;
 
-public enum GeneratorStage {
-	TERRAIN,
-	SURFACE,
-	FEATURES,
-	LIGHTING,
-	POPULATION,
-	LIVE;
+public class GeneratorStage {
 
-	public static GeneratorStage getFirstStage() {
-		return values()[0];
+	public static GeneratorStage LIVE = new GeneratorStage("LIVE");
+	
+	
+	private final String name;
+	
+	private byte ordinal = 0;
```

Comment:
I think I said that previously, when writing to disk - String should be used. When sending to client - client doesn't need to know the stage.